people-and-planet-ai/weather-forecasting/visualize.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utility functions to visualize data.
+
+Color names from https://chir.ag/projects/name-that-color
+"""
+
+from __future__ import annotations
+
+import numpy as np
+import plotly.graph_objects as go
```

Comment:
Not required for this PR (since it's time sensitive) but consider using `graph_object` rather than the shortened qualifier for better readability in the future.

Reply:
We still need to wait until the tests run so I just went ahead and changed it. The formatting introduced more line breaks to keep lines short, but it's not a big deal. The linter seems to be happy with the new changes.